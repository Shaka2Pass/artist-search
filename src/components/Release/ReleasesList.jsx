import Release from '../Release/Release.jsx';
import withList from '../withList';

const ReleasesList = withList(Release, 'releaseResults');

export default ReleasesList;
