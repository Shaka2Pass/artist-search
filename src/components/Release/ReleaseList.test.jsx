import React from 'react';
import { shallow } from 'enzyme';
import ReleasesList from '../Release/ReleasesList.jsx';

describe('ReleaseList component', () => {
  it('matches the ReleaseLists component', () =>{
    const releaseResults = [
      {
        title: 'Gloria',
        image: 'http://coverartarchive.org/release/03be9452-549a-4f30-a96f-44f794d50253/front',
        id: '123'
      },
      {
        title: 'Gloria',
        image: 'http://coverartarchive.org/release/03be9452-549a-4f30-a96f-44f794d50253/front',
        id: '123'
      }
    ];

    const wrapper = shallow(<ReleasesList releaseResults={releaseResults} />);
    expect(wrapper).toMatchSnapshot();
  });
});
