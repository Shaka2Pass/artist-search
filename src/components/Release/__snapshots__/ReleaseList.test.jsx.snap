// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReleaseList component matches the ReleaseLists component 1`] = `
<ol>
  <li
    key="123"
  >
    <Release
      id="123"
      image="http://coverartarchive.org/release/03be9452-549a-4f30-a96f-44f794d50253/front"
      title="Gloria"
    />
  </li>
  <li
    key="123"
  >
    <Release
      id="123"
      image="http://coverartarchive.org/release/03be9452-549a-4f30-a96f-44f794d50253/front"
      title="Gloria"
    />
  </li>
</ol>
`;
